bpf/bpf_sock.c

```diff
@@ -345,6 +347,53 @@ static __always_inline int __sock4_xlate_fwd(struct bpf_sock_addr *ctx,
 		return -ENOENT;
 	}
 
+    /* Service translation logic for a local-redirect service can cause
```

Comment:
Nit: s/local-redirect/local-policy-redirect/

Reply:
`local-redirect` _is_ the service type.

Reply:
Aha, I thought that the LRP service type is called `local-redirect-policy`.